docs/user_guide/creation/GeoDistanceTransformer.rst

```diff
@@ -0,0 +1,169 @@
+.. _geo_distance_transformer:
+
+.. currentmodule:: feature_engine.creation
```

Comment:
We also need to add corresponding files to `docs/api/..` so that the documentation is created from the class docstrings.